egs/wsj/s5/utils/validate_text.pl

```diff
@@ -103,16 +93,17 @@ sub check_allowed_whitespace {
   my $file = shift;
   my $filename = shift;
   my $pos = tell($file);
-  (my $is_utf, my @lines) = get_utf8_or_bytestream($file);
+  (my $is_utf, my @lines) = get_unicode_stream($file);
   seek($file, $pos, SEEK_SET);
   if ($is_utf) {
     my $has_invalid_whitespaces = validate_utf8_whitespaces(\@lines);
     if ($has_invalid_whitespaces) {
       print STDERR "$0: ERROR: text file '$filename' contains disallowed UTF-8 whitespace character(s)\n";
       return 0;
     }
+    return 1;
   }
-  return 1;
+  return 0;
```

Comment:
i think this change may not be right.. it does `check_allowed_whitespace(..) or exit 1`.

Reply:
IMO the change is right -- if the text is not UTF-or-ASCII, we have to return false, so that ' or exit 1' gets executed